aion/test/lib/ranking_test.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Aion.RankingTest do
+  use Aion.ModelCase
+  alias Aion.{Repo, User, UserCategoryScore, Ranking}
+
+  describe "Ranking.general" do
+    test "returns scored points in each category for each user" do
+      user1 = Repo.insert! User.registration_changeset(%User{}, %{name: "John", email: "test1@example.com", password: "test123"})
+      user2 = Repo.insert! User.registration_changeset(%User{}, %{name: "Michael", email: "test2@example.com", password: "test123"})
+      Repo.insert! %UserCategoryScore{user: user1, score: 2}
+      Repo.insert! %UserCategoryScore{user: user1, score: 4}
+      Repo.insert! %UserCategoryScore{user: user2, score: 5}
+
+      result = Ranking.general()
+
+      assert [%{user_name: "John", score: 6}, %{user_name: "Michael", score: 5}] = result
```

Comment:
I'm not sure if this is the right way to assert. I mean, you don't need the `assert` keyword here at all, as assert will be okay with anything that is truthy here. 

If the pattern matching doesn't work, the error will be raised before the assert is evaluated anyway.

You can either switch that to `== result` or you can remove the assert statement or you can export the values and check them in consecutive rows. (but I think the first one is the clearest solution)

Reply:
+ I think its more intuitive to have value that is being asserted on the left, so `result == expected_value`.

Reply:
Agreed